packages/typography/README.md

```diff
@@ -0,0 +1,70 @@
+# Typography
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/typography.svg)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/typography
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/typography
+```
+
+# H1
+
+## Example
+
+```jsx
+import { H1, H2, Subtitle, Body, InlineCode, Disclaimer, Overline } from '@leafygreen-ui/typography';
+<H1>Heading 1</H1>
+<H2>Heading 2</H2>
+<Subtitle>Subtitle</Subtitle>
+<Body weight="medium">Body</Body>
+<InlineCode>Code</InlineCode>
+<Disclaimer>Disclaimer</Disclaimer>
+<Overline>Overline</Overline>
+```
+
+**Output HTML**
+
+```html
+<h1 class="leafygreen-ui-wbskfk">Heading 1</h1>
+<h2 class="leafygreen-ui-1t0mh6j">Heading 2</h2>
+<h6 class="leafygreen-ui-1dmxpt6">Subtitle</h6>
+<p class="leafygreen-ui-wkgw79">Body</p>
+<code class="leafygreen-ui-18bk0d8">Code</code>
+<small class="leafygreen-ui-1cggyhz">Disclaimer</small>
+<div class="leafygreen-ui-vezyzr">Overline</div>
+```
+
+## Properties
+
+All props extend the HTMLElementProps of their root tag, however `<Body />` and `<Overline/>` accept extra props
+
+| Component    | Root    |
+| ------------ | ------- |
+| `H1`         | `h1`    |
+| `H2`         | `h2`    |
+| `Subtitle`   | `h6`    |
+| `Body`       | `p`     |
+| `InlineCode` | `code`  |
+| `Disclaimer` | `small` |
```

Comment:
I'm wondering about this one given it's pretty narrow. Might consider just calling this `Small` instead.

Up to you on this one!

Reply:
Happy leaving as Disclaimer bc it advises on usecase better (I didn't know what small was/that it was an HTML tag) but up to you, I don't feel too strongly about it